CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   + Wrap only objects (i.e. not primitive types or arrays) returned by custom handlers in arrays in create, read, and update resolvers
   + Delete mutations return the length of an array that is returned by a `DELETE` custom handler or 1 if a single object is returned
 - Don't generate fields for key elements in update input objects
+- Allow services that are not instances of `cds.ApplicationService`
```

Comment:
Should we mention that the developer needs to provide the correct set of services if using the API directly?

Reply:
Ah yes that would be helpful. Or is there some sort of programatic check that we can still perform?